test/libethereum/blockchain.cpp

```diff
@@ -124,632 +85,425 @@ void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 				else
 					importBlockNumber++;
 
-				//each time construct a new blockchain up to importBlockNumber (to generate next block header)
-				vBiBlocks.clear();
-				vBiBlocks.push_back(biGenesisBlock);
-
-				TransientDirectory tdBc;
-				TransientDirectory tdStateDB;
-				FullBlockChain<Ethash> bc(rlpGenesisBlock.out(), AccountMap(), tdBc.path(), WithExisting::Kill);
-
-				//OverlayDB database (State::openDB(td_stateDB.path(), h256{}, WithExisting::Kill));
-				State state(OverlayDB(State::openDB(tdStateDB.path(), h256{}, WithExisting::Kill)), BaseState::Empty); //= importer.m_statePre;
-				ImportTest::importState(o["pre"].get_obj(), state);
-				state.commit();
-
-				//import previous blocks
-				for (size_t i = 1; i < importBlockNumber; i++) //0 block is genesis
+				if ((string)boost::unit_test::framework::current_test_case().p_name != "bcWalletTest")
```

Comment:
that would be a trouble. cause I keep history of all imported blocks with it states
thus blockchain could be rebuld from any block for the test
changes might require to reconsider this logic while quicker solution is add some test option to disable it.  
but if there will be tests with blockchain overwrite at block like #100  this would be a problem